code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -39,6 +39,14 @@
 		r_store,
 		w_uniform)
 
+/mob/living/carbon/human/get_nonclothing_items() //pockets, hands, belt

```

Comment:
It's more efficient but more redundant, and therefore harder to maintain.
The cost isn't that much higher, and it's not like inventory code is called every tick.